js/index.js

```diff
@@ -56,28 +56,62 @@ messageForm.addEventListener("submit", function (event) {
 
   newMessage.appendChild(removeButton);
 
-  const editButton = document.createElement('button');
-  editButton.innerText = 'Edit';
-  editButton.type = 'button';
+  const editButton = document.createElement("button");
+  editButton.innerText = "Edit";
+  editButton.type = "button";
 
-  editButton.addEventListener('click', function(){
-    const currentMsg = newMessage.querySelector('span').innerText;
-    const newMsgTxt = prompt('Enter your new message', currentMsg);
+  editButton.addEventListener("click", function () {
+    const currentMsg = newMessage.querySelector("span").innerText;
+    const newMsgTxt = prompt("Enter your new message", currentMsg);
 
-    if(newMsgTxt !== null){
-        newMessage.querySelector('span').innerText = newMsgTxt;
+    if (newMsgTxt !== null) {
+      newMessage.querySelector("span").innerText = newMsgTxt;
     }
   });
 
   newMessage.appendChild(editButton);
 
   messageList.appendChild(newMessage);
 
-  if(messageList.children.length === 0){
-    messageSection.style.display = 'none';
+  if (messageList.children.length === 0) {
+    messageSection.style.display = "none";
   } else {
-    messageSection.style.display = 'block';
+    messageSection.style.display = "block";
   }
-
-
 });
+
+const username = "Gayana-G";
+// Creating a fetch
+fetch(`https://api.github.com/users/${username}/repos`)
+  .then((response) => {
+    if (response.ok) {
+      return response.text();
+    } else {
+      throw new Error("Failed to fetch repositories");
+    }
+  })
+  .then((data) => {
+    const repositories = JSON.parse(data);
+    console.log(repositories);
+
+    // DOM Selection to select the projects section by id
+    const projectSection = document.getElementById("Projects");
+    // Query the projectSection to select the <ul> element
+    const projectList = projectSection.querySelector("ul");
+
+    for (let i = 0; i < repositories.length; i++) {
```

Comment:
Could use for...of here and eliminate the index (for (let repository of trepositories)).